packages/next/build/babel/preset.ts

```diff
@@ -113,7 +113,8 @@ export default (
     presetEnvConfig.targets = {
       // Targets the current process' version of Node. This requires apps be
       // built and deployed on the same version of Node.
-      node: 'current',
+      // This is the same as using "current" but explicit
+      node: process.versions.node,
```

Comment:
We probably need to put `process.versions.node` into `cache.version` to invalidate the cache when node.js version has changed. At least for the server bundle.

Reply:
Yeah I was thinking that as well, though it might cause some issues when running on multiple similar node versions. We'll likely want to make that change in Next.js 12.